.github/workflows/go.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           go-version: 1.13
 
       - name: Check out code into the Go module directory
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
```

Comment:
`checkout@v2` is faster and has more features than `checkout@v1`, such as easy pushing back to the repository